### DISCUSSION TITLE
Maybe a bug in 6.0.1 - any ideas?

### DISCUSSION THREAD

**Discussion by Nasicus** [cypress-io/cypress#9421] [Questions and Help]
Just upgraded to `6.0.1` and replaced all my `cy.route` with `cy.intercept`.

I think this is a bug, but didn't create one yet, since it happens only every ~3 times I run the test and I have no idea why or how to reproduce it.

![image](https://user-images.githubusercontent.com/1086065/100885777-51237280-34b3-11eb-9fe6-52726330b9e7.png)

As you can see it says it timed out waiting for the response, even though it even writes one line above `ResponseReceived`.

And then I rerun it and sometimes it suddenly works:
![image](https://user-images.githubusercontent.com/1086065/100885944-7f08b700-34b3-11eb-8b06-f17d8353b015.png)

Code looks like this:

```
cy.intercept('POST', '**/b2c_users/with_hash/hash').as('createUser');
// some other test code
// ...
 cy.wait('@createUser');
```

Note that we have in total around 2400 tests (not all of them use intercept and not all of theme are UI testes, some only test the endpoints). And only this one and another one (same file and same route) fails.

**Comment by lpanger** [comment]
I'm seeing the same thing and it usually happens with larger and/or slower API calls. Thought maybe there might be a way to intercept only `Completed` instead of `ResponseReceived` states but I haven't found anything.

Another maybe unrelated thing is that sometimes this `ResponseReceived` will have a child and you can expand it in the command log to see `Completed`.

**Reply by Nasicus** [reply]
Hmm interesting. So it's not only me.


@jennifer-shehane  sorry for mentioning directly.

Should I create a bug out of this? Note that I can still only reproduce it randomly.

**Comment by lpanger** [comment]
I think this is the bug - https://github.com/cypress-io/cypress/issues/8999. Looks like it's fixed in 6.1.0 but I haven't confirmed

**Reply by jennifer-shehane** [reply]
Yah, it does look like this bug, if anyone can confirm after upgrading to 6.1.0 this should be fixed.
